okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -1127,6 +1128,8 @@ public void data(boolean inFinished, int streamId, BufferedSource in, int length
 
         Buffer buf = new Buffer();
         buf.write(in.buffer(), length);
+        PerfMark.event("OkHttpClientTransport$ClientFrameHandler.data",
```

Comment:
From easy to use point of view, I recommend you also put events within a task.

Reply:
This is a recommendation. Events are displayed as thin lines in the output graph, they can be hard to find with out parent tasks.